python/daqconf/apps/dataflow_gen.py

```diff
@@ -30,6 +30,7 @@
 import dunedaq.appfwk.app as app # AddressedCmd,
 import dunedaq.dfmodules.triggerrecordbuilder as trb
 import dunedaq.dfmodules.datawriter as dw
+import dunedaq.hdf5libs.hdf5filelayout as h5fl
 import dunedaq.dfmodules.hdf5datastore as hdf5ds
```

Comment:
Is this import still needed?

Reply:
I believe so, in order to construct the file layout parameters in the configuration. This is then part of the hdf5datastore configuration.